examples/vector_indices/SimpleIndexDemo.ipynb

```diff
@@ -151,6 +151,73 @@
     "display(Markdown(f\"<b>{response}</b>\"))"
    ]
   },
+  {
```

Comment:
The notebook organization is a mess atm, but imo I'd create a new folder under `examples` called QueryTransformations which specifically demos HyDE.

Reply:
Added a separate notebook with HyDE examples.